package.json

```diff
@@ -71,14 +71,13 @@
     "url": "https://github.com/thepracticaldev/dev.to/issues"
   },
   "homepage": "https://github.com/thepracticaldev/dev.to#readme",
-  "dependencies": {
+  "devDependencies": {
     "@stoplight/spectral": "5.1.0",
     "@storybook/addon-actions": "3.4.12",
     "@storybook/addon-knobs": "3.4.12",
     "@storybook/addon-links": "3.4.12",
     "@storybook/addons": "3.4.12",
     "@storybook/react": "3.4.12",
-    "babel-core": "^7.0.0-bridge.0",
```

Comment:
This package is not required as scoped packages are used for Babel 7+ now, i.e. @babel/core. I believe this package was used in the interim pre-@babel/core. I used the `babel-upgrade` package via `npx babel-upgrade --write` and this was added.

It is also the only package that was added to the `devDependencies` section of package.json within the last day.

My assumption is this will deploy fine on Heroku now, but 🤞

Reply:
Hmm jest test are failing now. Will need to investigate in the AM